pkg/stringid/stringid.go

```diff
@@ -4,19 +4,27 @@ import (
 	"crypto/rand"
 	"encoding/hex"
 	"io"
+	"regexp"
 	"strconv"
 )
 
+const shortLen = 12
+
+// Determine if an arbitrary string *looks like* a short ID.
+func IsShortID(id string) bool {
+	return regexp.MustCompile("^[a-z0-9]{12}$").MatchString(id)
```

Comment:
I don't think this is right. Given the following:

```
docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
444dc9db8345        busybox:latest      "top"               4 seconds ago       Up 4 seconds                            stupefied_colden
```

I can simply type `docker logs 4` without ambiguity: the 12 short ID fixed length seems arbitrary there. And as *anything* can be a short ID, I'm unfortunately not sure that the purpose of the PR remains.